freqtrade/exchange/exchange.py

```diff
@@ -1528,7 +1528,7 @@ def refresh_latest_ohlcv(self, pair_list: ListPairsWithTimeframes, *,
         :return: Dict of [{(pair, timeframe): Dataframe}]
         """
         logger.debug("Refreshing candle (OHLCV) data for %d pairs", len(pair_list))
-        # TODO-lev: maybe depend this on candle type?
+        # TODO: maybe depend this on candle type?
```

Comment:
i think this can be removed completely.
this method IS now depending on candle_type ... if it means something else, i don't understand the comment.